scripts/rabbitmq.sh

```diff
@@ -3,11 +3,16 @@
 echo ">>> Installing RabbitMQ"
 
 apt-get -y install erlang-nox
-wget http://www.rabbitmq.com/rabbitmq-signing-key-public.asc
-apt-key add rabbitmq-signing-key-public.asc
+wget http://www.rabbitmq.com/rabbitmq-release-signing-key.asc
+apt-key add rabbitmq-release-signing-key.asc
 echo "deb http://www.rabbitmq.com/debian/ testing main" > /etc/apt/sources.list.d/rabbitmq.list
 apt-get update
-apt-get install rabbitmq-server
+apt-get install -y rabbitmq-server
+rabbitmq-plugins enable rabbitmq_management
+
+# Allow guests to login thru rabbitmq management
+echo "[{rabbit, [{loopback_users, []}]}]." >> /etc/rabbitmq/rabbitmq.config
```

Comment:
Why allow guest users?  In Vagrantfile you have a rabbitMQ user and password created which is more secure - you can safely remove this line.